webpack/scenes/ModuleStreams/Details/__tests__/ModuleStreamDetailsActions.test.js

```diff
@@ -0,0 +1,42 @@
+import { testActionSnapshotWithFixtures } from '../../../../move_to_pf/test-utils/testHelpers';
+import api from '../../../../services/api';
+import { apiError } from '../../../../move_to_foreman/common/helpers';
+import { loadModuleStreamDetails } from '../ModuleStreamDetailsActions';
+import { details } from './moduleStreamDetails.fixtures';
+
+jest.mock('../../../../services/api');
+jest.mock('../../../../move_to_foreman/common/helpers');
+
+const fixtures = {
+  'should load module stream details on success': {
+    action: () => loadModuleStreamDetails('1'),
+    test: () => {
+      expect(api.get.mock.calls).toMatchSnapshot();
+      expect(apiError).not.toHaveBeenCalled();
+    },
+  },
+  'should load fail on bad api call': () => (dispatch) => {
+    api.get.mockImplementation(async () => {
```

Comment:
personally I don't find `api.get.mockImplementation` essential.
for testing the api itself it's fair, but for unit tests it's overkill. I prefer to mock the api module with a promise which returns a response.

Reply:
Is there a good example of this?

Reply:
sorry, I thought you used [mockRequest](https://github.com/Katello/katello/blob/master/webpack/mockRequest.js), `mockImplementation` is exactly what I meant for :)